### DISCUSSION TITLE
Should I add a new pull request for a graph plugin ?

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#881] [Q&A] [answered]
Actually, I am going to develop or use some other graph plugins for my docs, should I contribute it as a integration to the ui integrations ?

**Comment by fuma-nama** [comment]
Can't get what you mean, could you elaborate more?

**Comment by Gitstar-OC** [comment]
In the integration of the fumadocs, I was thinking adding this library for the developers who want to use graphs for maths or various tasks in mdx. https://recharts.org/en-US/. It would be helpful if you added graphs in integrations for developers.

**Reply by Gitstar-OC** [reply]
Like this one https://fumadocs.vercel.app/docs/ui/math

**Reply by fuma-nama** [reply, marked as answer]
Just use it like a normal component, there’s no need for an integration for that
